package.json

```diff
@@ -11,9 +11,7 @@
     "author": "Cloudify Platform Ltd. <cosmo-admin@cloudify.co>",
     "scripts": {
         "clean": "git clean -fXd -e \\!node_modules -e \\!node_modules/**/*",
-        "publish:major": "bash ci/publish-version.sh major",
-        "publish:minor": "bash ci/publish-version.sh minor",
-        "publish:patch": "bash ci/publish-version.sh patch"
+        "publish": "bash packages/scripts/create-version.sh"
```

Comment:
I think it's OK to simplify it and rely on user.
It's easy to use as if you call `npm run publish` you'll see help description for `create-version.sh` script.